github/users.go

```diff
@@ -144,6 +144,22 @@ type UserListOptions struct {
 
 // ListAll lists all GitHub users.
 //
+// To paginate through all users, populate 'Since' with the ID of the last user.
+// Example (warning: this will quickly eat through your GitHub API quota!):
+//
+//      opts := &github.UserListOptions{}
+//      for {
+//        users, _, err := client.Users.ListAll(opts)
+//        if err != nil {
+//          log.Fatalf("error listing users: %v", err)
+//        }
+//        if len(users) == 0 {
+//          break
+//        }
+//        opts.Since = *users[len(users)-1].ID
+//        // Process users...
+//      }
```

Comment:
I would suggest making this Go code `gofmt`ed. Right now it uses 1 space indent, followed by 2 spaces. How about this:

https://gist.githubusercontent.com/shurcooL/43a5922765e1918ae00f29f315e9b41c/raw/e1de820beeece8b3a6a881e8f0d1913901e7c71c/comment.go

(Note that the comment always begins with 2 slashes and a space. The rest is its content. The code section is indented by 1 tab. This is the most idiomatic and consistent way to write Go comments, at least if you're as picky about them as me.)